scripts/bert/finetune_classifier.py

```diff
@@ -135,6 +134,25 @@
     'for both bert_24_1024_16 and bert_12_768_12.'
     '\'wiki_cn\', \'wiki_multilingual\' and \'wiki_multilingual_cased\' for bert_12_768_12 only.'
 )
+parser.add_argument(
+    '--pretrained_bert_parameters',
+    type=str,
+    default=None,
+    help='Pre-trained bert model parameter file. default is None'
+)
+parser.add_argument(
+    '--model_parameters',
+    type=str,
+    default=None,
+    help='Model parameter file. default is None'
```

Comment:
Also explain that this file is loaded at the beginning of the program for inference/training (not output produced by the program).